Remove istio specific annotations from deployments (#82)
kyma-project/examples: gateway/README.md

- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version 1.10.0
- A token fetched from the Console UI which is later referred to as **\{token\}**. For more details, see the **NOTE** in the **Fetch token** section.

#### Create a service

>**NOTE:** Almost all steps in this tutorial refer to a lambda but you can apply them also to the service Deployment by changing the directory and names in some places.

> 1. Export your Environment as variable by replacing the `{environment}` placeholder in the following command and running it:

Reviewer: ```suggestion
1. Export your Environment as a variable. Replace the `{environment}` placeholder in the following command and run it:
```
